app/controllers/api/v1/sops_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api::V1
+  class SopsController < ApiController
+    def index
+      keyword, from, size = sop_search_params
```

Comment:
multi-field assignment is not a discipline use, please use protected_params to return a hash of search_fields instead